### PR TITLE
add shared responsibility model page

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new Shared Responsibility Model page.
* Added image.
* Fixed max-width for images.

[EDU-2060]: https://aziontech.atlassian.net/browse/EDU-2060?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+NIST Special Publication 500-292 defines five essential characteristics of cloud services:
+
+- On demand self-service 
+- Broad network access 
+- Resource pooling 
+- Rapid elasticity 
+- Measured service
+
+The publication also provides a simple and unambiguous categorisation of cloud services models:
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx:25]:
```suggestion
The publication also provides a simple and unambiguous categorization of cloud services models:
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+- Cloud software as a service (SaaS)
+- Cloud platform as a service (PaaS)
+- Cloud infrastructure as a service (IaaS)
+
+Azion’s Edge Platform does not accurately fit into any of the three delivery models and can be described as a blend of PaaS and SaaS models.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx:31]:
```suggestion
Azion’s Edge Platform doesn't accurately fit into any of the three delivery models and can be described as a blend of PaaS and SaaS models.
```

File: src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+---
+
+## Shared Responsibility Model
+
+The shared responsibility model is central to understanding the roles of the customers and Azion in the context of application security and the use of Azion Edge Platform.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx:37]:
```suggestion
The Shared Responsibility Model is central to understanding the roles of the customers and Azion in the context of application security and the use of Azion Edge Platform.
```

File: src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Customers are responsible for the correct configuration and usage of the controls and for the application and content they put on or through Azion’s Edge Platform.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx:39]:
```suggestion
Customers are responsible for the correct configuration and usage of the controls and for the application and content they put on or through Azion Edge Platform.
```

File: src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+- Configuration change control.
+- Managing their data, such as content and application and source code (including encryption options).
+- Identity and access management, including user provisioning and their roles.
+- Monitoring quotas and limits.
+- Monitoring consumption and billing.
+- Correct usage of Azion’s platform.
+
+Customers are also responsible for:
+
+- Adhering to Azion’s Acceptable Use Policy and all applicable laws and regulations.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx:82]:
```suggestion
- Adhering to [Azion’s Acceptable Use Policy](/en/documentation/agreements/acceptable-use-policy/) and all applicable laws and regulations.
```

**Review by MarianaAguilera — APPROVED** [review]
style and a suggestion

File: src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx [hunk 6]
```diff
@@ -0,0 +1,97 @@
+---
+title: Shared Responsibility Model
+description: This document highlights the roles and responsibilities of each party when organizations use the Azion Edge Platform.
+meta_tags: azion, shared responsibility,
```
Comment (bru-andrade) [inline src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx:4]:
```suggestion
meta_tags: azion, shared responsibility
```

**Pull request merged**